Jasmine/Common/ViewControllers/GameElements/SquareGridViewController.swift

```diff
@@ -32,7 +32,7 @@ class SquareGridViewController: UIViewController {
     }
 
     /// Stores the database that is used to display onto the collection view in this view controller.
-    fileprivate var collectionData: [Coordinate: String] = [:]
+    fileprivate var collectionData: CharacterGrid!
```

Comment:
Soo, it's okay for VC to hold the model ADT, instead of a `Coordinate: String` map?